db.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require('mongoose')
+
+mongoose.connect(
+  process.env.MONGODB_URI || 'mongodb+srv://dineshbukya:FOkzrIVQvxtsygRh@cluster0.b9nwo.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0',
```

Comment:
For this lab, you should have been able to use the DB settings as defined in the video and the readme: 

```js
// db.js
const mongoose = require('mongoose')

/**
 * In this example we are connecting to a local MongoDB instance. This instance is running via docker-compose in our GitHub Codespaces environment.
 * In a real-world application, you would want to use a cloud-based MongoDB service like MongoDB Atlas.
 */
mongoose.connect(
  process.env.MONGODB_URI || 'mongodb://root:example@localhost:27017/?authSource=admin',
  {
    useNewUrlParser: true,
    useUnifiedTopology: true
  }
)

module.exports = mongoose
```

Last semester we used the Atlas, but we simplified it for this term